src/ocl/convolutionocl.cpp

```diff
@@ -566,10 +566,6 @@ void ValidateConvTensors(const ConvTensors& tensors)
     const auto trivial_tensor_types_not_matched =
         tensors.xDesc.GetType() != tensors.yDesc.GetType() &&
         tensors.xDesc.GetType() != miopenInt8 && tensors.xDesc.GetType() != miopenInt8x4;
-    const auto int8_in8x4_tensor_not_matched =
```

Comment:
[Q] This has been added #1572, see https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1572/files#r955487952. I am curious, why this is not necessary anymore?

Reply:
These checks make assumptions about the int8 implementation of various solvers which is best left to the `IsApplicable` methods in those solvers. By putting such checks here we are making sweeping assumptions about the capability of the whole libarary. For instance in this case the assumption is that if the input and the output tensor's data type is not the same, the convolution config is invalid. Which is not true. Or the only possible output data type if input is int8 is int8x4. Either of which may be incorrect based on different solvers that now exist in MIOpen.

Reply:
@JehandadKhan To be honest, I always felt some bad smell there ;) Resolved.